docs/en/observability/create-alerts.asciidoc

```diff
@@ -3,39 +3,66 @@
 
 [IMPORTANT]
 ====
-Ensure Alerting is already set up in {kib}. For details, see
+Make sure alerting is already set up in {kib}. For details, see
 {kibana-ref}/alerting-setup.html[Setup and prerequisites].
 ====
 
-Alerting enables you to detect complex *conditions* defined by a rule
+Alerting enables you to detect complex conditions defined by a *rule*
 within the Logs, Metrics, Uptime, and APM apps. When a condition is met, the rule
 tracks it as an *alert* and responds by triggering one or more *actions*.
 
-Listed on the <<view-observability-alerts,Alerts>> page are all of your
-observability alerts that have met a condition defined by a rule you created
-using the Logs, Metrics, Uptime, or APM apps.
+Alerts related to {observability} apps including Logs, Metrics, Uptime, and APM can be managed in the {observability} UI.
```

Comment:
@colleenmcginnis With the addition of the brand new Obsevability Rule Management page, I would mention that not only alerts but `rules` as well can now be managed in the observability UI. This is what I am suggesting:
```suggestion
Alerts and rules related to {observability} apps including Logs, Metrics, Uptime, and APM can be managed in the {observability} UI.
```